packages/thumbprint-global-css/package.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "@thumbtack/thumbprint-global-css",
+    "description": "Global reset, body, and anchor styling for building Thumbtack UI",
+    "version": "0.1.0",
```

```suggestion
    "version": "0.0.0",
```
The first Lerna release will ask what version to use and then we choose `0.1.0`. Otherwise Lerna will think that the package is already released at `0.1.0`.